### DISCUSSION TITLE
关于intel核显硬解

### DISCUSSION THREAD

**Discussion by kli2013** [hooke007/mpv_PlayKit#268] [Q&A 问答]
```
hwdec = d3d11va-copy
gpu-context = auto
```
```
hwdec = auto-copy
gpu-context = auto
```
都非常卡
去掉-copy才正常

intel核显不支持copy吗?

**Comment by kli2013** [comment]
测试的是av1的8k

普通264和265好像可以dxva2-copy

**Comment by hooke007** [comment]
这种脱离现实用途的片源就不要copy了，copy只是折磨和浪费带宽。本身核显配的内存就很可怜了（

**Reply by kli2013** [reply]
好的,请问另外一个问题,怎么指定核显运行mpv呢?
这样我就可以用auto的配置来单独解码8k
其他的依然用独显

**Reply by hooke007** [reply]
不是有个device指定设备吗，但我没试过能不能运行时变动

**Reply by kli2013** [reply]
指定成功,不过我不会整体切换配置
现在想要直接解码8k就必须hwdec =auto
然后想copy的时候快捷键切换为nvdec-copy

**Reply by hooke007** [reply]
了解profiles.conf的写法

**Comment by kli2013** [comment]
profiles.conf里添加

[8kintel]
 profile-desc = 8kintel
 profile-cond = width >= 7000
 profile-restore = copy
 hwdec = auto
 d3d11-adapter = "Intel(R) UHD Graphics 730"
怎么不能自动切换啊?

 hwdec = auto
 d3d11-adapter = "Intel(R) UHD Graphics 730"
在mpv.conf里改了是能正常核显解码的

`开启控制台运行
apply-profile 8kintel
也没反应

**Reply by hooke007** [reply]
不是所有属性都能在运行时更改

**Reply by kli2013** [reply]
好的
